plugins/gpio/meson.build

```diff
@@ -1,10 +1,7 @@
-if get_option('plugin_gpio')
+if get_option('plugin_gpio').enabled()
```

Comment:
Is this how you're supposed to use autofeatures? I think when it's set to auto then `enabled()` is going to false, no? I thought the way you were supposed to use it by setting a variable inside the cc.has_header_symbol conditional block? On that, maybe the gpio.h check should move into the gpio plugin directory?

Reply:
Oh I see what you mean, it should be `.auto()`.

Yeah I think since this is only plugin using it moving the header check here is reasonable.